src/MapzenPlaceConfig.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  MapzenPlaceConfig.swift
+//  Pods
+//
+//  Created by Sarah Lensing on 3/20/17.
+//
+//
+
+import Foundation
+import Pelias
+
+public class MapzenPlaceConfig : NSObject {
```

Comment:
What are you thoughts on moving all of these into a single swift file? It's common practice to have common structs / class grouped in the same file in Swift.